packages/ts-config-webpack-plugin/test/TsConfigWebpackPlugin.test.js

```diff
@@ -33,7 +33,7 @@ describe('TsConfigWebpackPlugin standalone', () => {
 		const options = {};
 		const instance = new TsConfigWebpackPlugin(options);
 		const resolvedPath = instance.resolveTsConfigFilePath(__dirname);
-		expect(resolvedPath).toEqual(path.resolve(__dirname, '../tsconfig.json'));
+		expect(resolvedPath).toEqual(path.resolve(__dirname, path.join('..', 'tsconfig.json')).replace(/\\/g, '/'));
```

Comment:
what about `path.posix.resolve(__dirname, '../tsconfig.json'));`?

https://nodejs.org/api/path.html#path_path_posix

Reply:
See my other reply. Also the separator in the string can be problematic.